quality_control_stock_oca/models/stock_picking_type.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2014 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockPickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    def _create_qc_trigger(self):
+        for picking_type in self:
+            qc_trigger = {
+                "name": picking_type.name,
```

Comment:
```suggestion
                "name": picking_type.name,
```

if we have 2 warehouse it will show same name. Is it posible to add hook function or change it to uniqe?

Reply:
Yes, what do you think about using `display_name` instead of `name` for `picking_type`? It'll be shown e.g. "Warehouse: operation"

Reply:
@dalonsod I agree to use `display_name`

Reply:
Done! Could you approve this? 😇